src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -370,6 +383,60 @@ module Types = struct
             ~resolve:(fun _ {Transaction_snark_work.Info.work_ids; _} ->
               One_or_two.to_list work_ids ) ] )
 
+  let signed_fee =
+    obj "SignedFee" ~doc:"Signed fee" ~fields:(fun _ ->
+        [ field "sign" ~typ:(non_null string) ~doc:"+/-"
+            ~args:Arg.[]
+            ~resolve:(fun _ fee ->
+              if Sgn.equal (Currency.Fee.Signed.sgn fee) Sgn.Pos then "+"
+              else "-" )
+        ; field "fee" ~typ:(non_null uint64) ~doc:"Fee"
```

Comment:
We could maybe call it feeMagnitude to help avoid people accidentally thinking it contains the sign?